### ISSUE TITLE
fit_portrait does not work

### ISSUE THREAD

**Issue by pljspahn** [buildinternet/supersized#111]
while using fit_portrait, the image still gets cropped on the top/bottom at certain window sizes.

**Comment by blakeperdue** [comment]
Agreed, doesn't work for me on OS X 10.8.2 Chrome Version 23.0.1271.95
